available-components.md

```diff
@@ -33,6 +33,10 @@ Noodle Bar facilitates setting up dataspaces that follow certain principles, ser
 
 This is official iSHARE library which provides core functionality for service consumers. In other words, it encapsulates HTTP request calls towards iSHARE endpoints, JWT response validation and mapping business logic.
 
+### [Postman Collections](https://dev.ishare.eu/demo-and-testing/postman.html)
+
+This link provides access to a set of tools called Postman collections, which are used to manually test how iSHARE APIs work within the BDI architecture. If you’re new to Postman, think of it as an application that lets you simulate how different software systems communicate with each other by sending and receiving data through APIs. In these collections, you’ll be pretending to be a company called “ABC Trucking,” using a special digital key that allows you to interact with the system as if you were that company. This is a safe way to try out the APIs and see how they work.
```

Comment:
"If you’re new to Postman, think of it as an application that lets you simulate how different software systems communicate with each other by sending and receiving data through APIs."

Dat kan echt beter....